util.py

```diff
@@ -51,43 +50,96 @@ def find_closest_servers(self, location, servers_to_search = None):
                 raise ValueError('Server <' + server + '> latitude/longitude could not be found!')
             server_location = geopy.Point(server_lat_lon[0], server_lat_lon[1])
             server_dict['distance'] = great_circle(item_location, server_location).km
-
             best_servers.append(server_dict)
-
         best_servers.sort(key=self.get_distance_key)
-
     return best_servers
 
+# Converts the local hostname to the simulation ip address
+#
+# params:
+#   local_host: the local hostname
+def convert_to_simulation_ip(local_host):
```

Comment:
yeah... i was debating about this yesterday with myself because `localhost` is kind of misleading to me. May be `local_ip`?